File: netlexapiwebadmin/netlexapiwebadmin/Models/FavorMovie.cs
using System;
using System.Collections.Generic;

namespace netlexapiwebadmin.Models
{
    public partial class FavorMovie
    {
        public int FavorId { get; set; }
        public int? MovieId { get; set; }
        public int? UserId { get; set; }

        public virtual Movie? Movie { get; set; }
        public virtual User? User { get; set; }
    }
}

File: lib/data/favormovie.dart
class FavorMovies {
  int? favorId;
  int? movieId;
  int? userId;
  Null movie;
  Null user;

  FavorMovies({this.favorId, this.movieId, this.userId, this.movie, this.user});

  FavorMovies.fromJson(Map<String, dynamic> json) {
    favorId = json['favorId'];
    movieId = json['movieId'];
    userId = json['userId'];
    movie = json['movie'];
    user = json['user'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['movieId'] = this.movieId;
    data['userId'] = this.userId;
    return data;
  }
}